src/main/java/org/sagebionetworks/bridge/services/AssessmentResourceService.java

```diff
@@ -145,7 +145,7 @@ public AssessmentResource updateSharedResource(String callerAppId, String assess
         checkArgument(isNotBlank(assessmentId));
         checkNotNull(resource);
         
-        Assessment assessment = assessmentService.getLatestAssessment(SHARED_APP_ID, assessmentId);
+        Assessment assessment = assessmentService.getLatestAssessment(SHARED_APP_ID, null, assessmentId);
```

Comment:
So why is ownerId not supplied here? Isn't it required for an update?

Reply:
Because it's a shared assessment with a compound ID, so we can't check it by ownerId. There's an AuthUtils check after this that does the correct permissions check.